.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+### Changelist
+[Describe or list the changes made in this PR]
+
+### Test Plan
+[Describe how this PR was tested (if applicable)]
+
+### Author/Reviewer Checklist
+- [ ] If this change affects functionality (features, bug fixes, breaking changes, etc.), update `protocol/CHANGELOG.md` and/or `indexer/CHANGELOG.md` appropriately.
+- [ ] If this PR has changes that result in a different app state given the same prior state and transaction list, manually add the `state-breaking` label.
+- [ ] If this PR isn't state-breaking but has changes that modify behavior in `PrepareProposal` or `ProcessProposal`, manually add the label `proposal-breaking`.
```

Comment:
Can a change be state-breaking and also a proposal-breaking change? I suppose we'd want to break those into two separate PRs?